documentation/docs/02-layouts.md

```diff
@@ -80,11 +80,14 @@ If an error component has a [`load`](#loading) function, it will be called with
 
 ```html
 <script context="module">
+	/**
+	 * @type {import('@sveltejs/kit').ErrorLoad}
+	 */
 	export function load({ error, status }) {
 		return {
 			props: {
-				title: `${status}: ${error.message}`
-			}
+				title: `${status}: ${error.message}`,
```

Comment:
I'm not sure I like the addition of the trailing commas on this line and the next

Reply:
removed them

thought i was using the default sveltekit prettier config, but now that i think of it i probably changed that setting